src/lib/temple/front/assets.ts

```diff
@@ -410,7 +413,10 @@ export function useFilteredSwapAssets(inputName: string = 'input') {
   const [searchValueDebounced] = useDebounce(tokenId ? toTokenSlug(searchValue, tokenId) : searchValue, 300);
 
   const filteredAssets = useMemo(
-    () => searchAssetsWithNoMeta(searchValueDebounced, assetSlugs, allTokensBaseMetadata, slug => slug),
+    () =>
+      searchAssetsWithNoMeta(searchValueDebounced, assetSlugs, allTokensBaseMetadata, slug => slug).sort(
+        assetsSortPredicate
+      ),
     [searchValueDebounced, assetSlugs, allTokensBaseMetadata]
```

Comment:
```suggestion
    [searchValueDebounced, assetSlugs, allTokensBaseMetadata, assetsSortPredicate]
```
The predicate changes as tokens balances and their exchange rates change